language-design.md

```diff
@@ -58,8 +134,20 @@ In what way is an API a language?
 What does the post on grayscale tell us about the process of API design?
 
 **Response**
-
-
+The greyscale post offers four options for functions that produce shades of grey. At
+the end, it includes a poll that asks readers to vote on the most intuitive one
+and also encourages users to leave a comment justifying their opinions [Verou, 2014].
+This shows that API design is very influenced by user feedback and is very concerned
+with how intuitive their functions are. Although some of the comments are not
```

Comment:
Did they do what the users want? I'm not sure that they did what the most popular option in the poll was...